app/controllers/users_controller.rb

```diff
@@ -1,19 +1,30 @@
 class UsersController < ApplicationController
 
-  def index
-
-  end
+  respond_to :html
 
   def show
-
+    @user = @current_user
```

Comment:
you can just access current_user method in any view in that app. so no need to populate that var